Formula/a/autobrr.rb

```diff
@@ -0,0 +1,77 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://autobrr.com/"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    system "npx", "pnpm", "install", "--dir", "web"
+    system "npx", "pnpm", "--dir", "web", "run", "build"
+
+    ldflags = "-s -w -X main.version=#{version} -X main.commit=#{tap.user}"
+
+    system "go", "build", *std_go_args(output: bin/"autobrr", ldflags: ldflags), "./cmd/autobrr"
+    system "go", "build", *std_go_args(output: bin/"autobrrctl", ldflags: ldflags), "./cmd/autobrrctl"
+  end
+
+  def post_install
+    (var/"autobrr").mkpath
+  end
+
+  service do
+    run [opt_bin/"autobrr", "--config", var/"autobrr/"]
+    keep_alive true
+    log_path var/"log/autobrr.log"
+  end
+
+  test do
+    require "socket"
+
+    assert_match version.to_s, shell_output("#{bin}/autobrrctl version")
+
+    server = TCPServer.new("127.0.0.1", 0)
+    free_port = server.addr[1]
+    server.close
```

Comment:
```suggestion
    port = free_port
```

Reply:
`free_port` is a Homerbrew test function, so you can remove the `TCPServer` part too.